### ISSUE TITLE
Question: Why is only one senator shown for an issue

### ISSUE THREAD

**Issue by badi** [5calls/5calls#71]
When I click on an issue and the text reads "Call this office" with the number of a senator or representative, how is the number and name determined? I only ever see one senator out of the two, and its always the democratic senator. Why are not both senators listed?

**Comment by jberman5** [comment]
I know if you click "Unavailable" at the bottom of the screen it will give you the option to call the other Senator, but it does seem to default to one over the other

**Comment by imaflamingo** [comment]
I only ever see the one representative for my district and one senator. I never see the other senator.

**Comment by imaflamingo** [comment]
I am seeing the other senator as of today

**Comment by jquense** [comment]
I also find the "need to click an action to get the next number" confusing. There should be an indication of progress like a "first of 4 people to call" bit

**Comment by SashaZd** [comment]
I think all relevant Senators with their phone numbers should be listed. That way we can call any of the ones voting Yes instead of No.

**Comment by nickoneill** [comment]
Yep, google civic API was not returning phone numbers for new senators until today so that should be fixed.

I *do* think we could be more clear on how many calls are part of an issue, this is only mentioned in tiny text on the bottom of an issue. Anyone want to make a proposal?

**Comment by beausmith** [comment]
Here is my proposal to solve:

- allowing user to know/view all the contacts
- allow user to switch to different active number (DC, local office number, etc)

![img_2359-small](https://cloud.githubusercontent.com/assets/28444/22621376/0c59c750-ead7-11e6-84cc-ef6b23b35ace.jpg)

cc @nickoneill @badi

**Comment by jquense** [comment]
:thumbsup: to the proposal. I'd be happy to try and implement if no one has grabbed it yet

**Comment by rebeccakaufman** [comment]
I think this is confusing. I want to keep things as simple as possible. 

Since we'll be contacting the same reps/senators on almost all issues, having a visual of them over and over might seem redundant.

I'm not an expert here.... but yea, how i feel.

**Comment by rebeccakaufman** [comment]
I also want to make a point about accessibility. A lot of our email feedback has been they like the site because of the simplicity. We definitely seem to be catching a demographic who are less internet/computer savvy so keeping their needs in mind is important to me.

**Comment by jquense** [comment]
simple is great! folks are are more noting that the current approach also confusing. Its not clear that there are gonna be more numbers and that you need to act to see them. anecdotally folks I've spoken to said they only called the first number asking that was it since it was the only one shown. I have a similar story my first time using the site

**Comment by jquense** [comment]
which is to say I think it's possible to maintain the current, straightforward guided approach while also making it easier for new folks to approach and being more useful for return usera

**Comment by beausmith** [comment]
@jquense - You're tapping into the same confusion which inspired this solution. If @nickoneill is open to it, I'd like to see it built… and once built put it in front of some peeps to gauge if it clears things up or makes things more confusing.

The select menu to change the active phone number should be done first as a separate feature.

**Comment by nickoneill** [comment]
I don't think this proposal is very clear on what action should be taken next. I suspect that some people don't make it to contact 2 or 3 because they don't know they should continue calling but we won't know exactly until some of the google analytics work I'm doing now starts returning data.

Keeping your time in mind, I would say hold off on this until we learn more. Or at least think about it a bit more on paper before you start committing it to code.
